README.md

```diff
@@ -11,4 +11,4 @@ This project is part of ASP.NET 5. You can find samples, documentation and getti
 
 ### Notes
 
-ASP.NET Security will not include Basic Authentication middleware due to its potential insecurity and performance problems. If you host under IIS you can enable it via IIS configuration. If you require Basic Authentication middleware for testing purposes or to use a database as a user source then please look at the samples from [leastprivilege](https://github.com/leastprivilege/BasicAuthentication.AspNet5) or [Kukkimonsuta](https://github.com/Kukkimonsuta/Odachi/tree/master/src/Odachi.Security.BasicAuthentication).
\ No newline at end of file
+ASP.NET Security will not include Basic Authentication middleware due to its potential insecurity and performance problems. If you host under IIS you can enable it via IIS configuration. If you require Basic Authentication middleware for testing purposes, as a shared secret authentication mechanism for server to server communication, or to use a database as a user source then please look at the samples from [leastprivilege](https://github.com/leastprivilege/BasicAuthentication.AspNet5) or [Kukkimonsuta](https://github.com/Kukkimonsuta/Odachi/tree/master/src/Odachi.Security.BasicAuthentication).
```

Comment:
Basic AuthN is how OAuth2 clients authenticate requests to the token endpoint - sure there are better ways - but that's how it is specified by the RFC.

So Basic Auth has its place - it is just def not for authenticating humans